TCAT/Controllers/RouteDetail+ContentViewController.swift

```diff
@@ -229,7 +233,7 @@ class RouteDetailContentViewController: UIViewController, GMSMapViewDelegate, CL
             self.banner!.dismissOnTap = true
             self.banner!.show(queuePosition: .front, on: navigationController)
             self.isBannerShown = true
-            UIApplication.shared.statusBarStyle = .lightContent
+            setNeedsStatusBarAppearanceUpdate()
```

Comment:
Could we put this in a didSet function for the isBannerShown variable? Not 100% sure if that works but if I understand what it does correctly, it could help us manage those changes without having to write this function everytime

Reply:
Yeah, good call, I don't know why I didn't do that. Ideally we should create a ViewController class with this functionality so we don't need to repeat it on every view, but this is a better step in that direction.

Reply:
Good find!